lib/routers/profile/permission.js

```diff
@@ -48,11 +48,19 @@ const removable = () => (req, res, next) => {
   };
   Profile.scopeSingle(params).get()
     .then(profile => {
-      const hasProjects = profile.projects.filter(project => project.status === 'active').length;
+      const hasAdditionalAvailbilityProjects = profile.projects.filter(p => {
+        return p.status === 'active' && p.additionalEstablishments.filter(ae => ae.id === req.establishment.id && ae.status === 'active').length;
+      }).length;
+
+      const hasProjects = profile.projects
+        .filter(project => project.establishmentId === req.establishment.id || project.additionalEstablishments.find(e => e.id === req.establishment.id && e.status === 'active'))
```

Comment:
```suggestion
        .filter(project => project.establishmentId === req.establishment.id)
```
The extra bit was redundant since you've pulled the `hasAdditionalAvailability` part out separately.

Reply:
ah right, that was a copy paste fail